monitor/upcloud.py

```diff
@@ -0,0 +1,68 @@
+from app.log import LOG
+from monitor.metric import UpcloudMetric, UpcloudMetrics, UpcloudRecord
+
+import base64
+import requests
+from typing import Any
+
+
+BASE_URL = "https://api.upcloud.com"
+
+
+def get_metric(json: Any, metric: str) -> UpcloudMetric:
```

Comment:
Nits: do we want to prefix these methods with _ to mark them as private?

Reply:
Marked as public because:
- They have tests, so we can easily import them
- In case we wanted to reuse them for any other upcloud metric in another exporter

Reply:
It's used in the tests. It's fine. If we need to use them from other place they don't touch any private thing. They are just converters.